images/win/scripts/Tests/Databases.Tests.ps1

```diff
@@ -3,7 +3,8 @@ Describe "MongoDB" {
         @{ ToolName = "mongo" }
         @{ ToolName = "mongod" }
     ) {
-        "$ToolName --version" | Should -ReturnZeroExitCode
+        $toolsetVersion = (Get-ToolsetContent).mongodb.default
+        ("$ToolName --version")[2].Split('"')[-2] | Should -BeLike "$toolsetVersion*"
```

Comment:
It's just a plain string without getting version.

Reply:
Should be fixed